fdbclient/Schemas.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Schemas.h
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2018 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FDBCLIENT_SCHEMAS_H
+#define FDBCLIENT_SCHEMAS_H
+
+#pragma once
+
+#include "flow/flow.h"
+#include "FDBTypes.h"
+
+extern const KeyRef statusSchema;
```

Comment:
I think I'd prefer these to not be globals like this, in part because usage of `statusSchema` in other files doesn't really appear as if the variable is defined as such. Maybe you could make them static members of a class?